spec/no_duplicate.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+
+git clone https://github.com/puppetlabs/puppetlabs-stdlib
+
+stdlib_funcs=`ls puppetlabs-stdlib/lib/puppet/parser/functions`
+extlib_funcs=`ls lib/puppet/parser/functions`
```

Comment:
I'm with @igalic on this one. You're looping over the strings anyway, might as well call `basename` then.